darwin/DarwinProcess.c

```diff
@@ -510,6 +524,7 @@ void DarwinProcess_scanThreads(DarwinProcess* dp, DarwinProcessTable* dpt) {
 
    vm_deallocate(mach_task_self(), (vm_address_t) thread_list, sizeof(thread_port_array_t) * thread_count);
    mach_port_deallocate(mach_task_self(), task);
+#endif
```

Comment:
Same here. If there's not even essential information filled in we can also just declare things fully incompatible with such old versions of MacOS …